infrasee/infrasee_core/models.py

```diff
@@ -2,4 +2,17 @@
 
 from django.db import models
 
-# Create your models here.
+
+class Port(models.Model):
+    name = models.CharField(max_length=256)
```

Comment:
what about using "description" instead of "name"? Then you could call out when its an application port vs an admin port.